typescript/packages/plugins/zilliqa/src/types.ts

```diff
@@ -0,0 +1,8 @@
+// Currently empty; needs to be kept separate from parameters.ts else
+// we end up with dependency resolution issues.
+
+// Slightly convoluted way to make {} mean empty; remove this when we
+// have some plugin parameters.
+const emptySymbol = Symbol("EmptyObject_type");
+
+export type ZilliqaPluginParams = { [emptySymbol]?: never };
```

Comment:
hmm you shouldn't need this. You can simply not pass anything to the Zilliqa plugin. The constructor of the plugin doesn't require anything

Reply:
Was there for expansion, and you're absolutely right. Removed.